elements/rh-avatar/rh-avatar.css

```diff
@@ -0,0 +1,112 @@
+:host {
+  display: inline-block;
+  width: min-content;
+}
+
+[hidden] {
+  display: none !important;
+}
+
+#container {
+  display: grid;
+  color: var(--rh-context-light-color-text-muted, #6a6e73);
+
+  --_colors: var(--rh-avatar-colors,
+  var(--rh-color-blue-200, #73bcf7)
+  var(--rh-color-cyan-300, #009596)
+  var(--rh-color-green-500, #3e8635)
+  var(--rh-color-red-300, #f56d6d)
+  var(--rh-color-purple-500, #6753ac));
+
+  column-gap: var(--rh-space-lg, 16px);
+  width: min-content;
+  grid-template-columns: min-content minmax(max-content, 250px);
+  grid-template-rows: min-content min-content;
+  grid-template-areas:
+    "a t"
+    "a s";
+}
+
+#title {
+  grid-area: t;
+  align-self: end;
+  font-weight: var(--rh-font-weight-heading-bold, 700);
+}
+
+#subtitle {
+  grid-area: s;
+  align-self: start;
+  font-weight: var(--rh-font-weight-heading-medium, 500);
+}
+
+:host([layout="block"]) #container,
+#container.mobile {
+  text-align: center;
+  place-items: center;
+  gap: 0;
+  grid-template-columns: minmax(max-content, 250px);
+  grid-template-areas: "a" "t" "s";
+  grid-template-rows:
+    minmax(var(--rh-avatar-size, var(--rh-size-icon-06, 64px)), var(--rh-size-icon-06, 64px))
+    min-content
+    auto;
+}
+
+:host([layout="block"]) :is(img, canvas),
+#container.mobile :is(img, canvas) {
+  margin-block-end: var(--rh-space-lg, 16px);
+}
+
+slot {
+  display: block;
+  max-width: 250px;
+}
+
+::slotted(a) { color: var(--rh-color-interactive-blue-darker, #0066cc); }
+::slotted(a:visited) { color: var(--rh-color-interactive-purple-darker, #6753ac); }
```

Comment:
Need interactive state for hover and active. Link color should turn `--rh-color-interactive-blue-darkest`